.storybook/themes/news.scss

```
@@ -1059,14 +1059,14 @@
 				small-manual-promo: (
 					components: (
 						media-item: (
-							flex: 1 0 min-content,
+							max-width: 33%,
```

This won't work on the "above"/"below" states... perhaps:
```suggestion
							flex-basis: 33%,
```
(untested)

---

ah nice -- that makes sense. I was only looking at the horizontal ones 🤦